smartrouter/image.yaml

```diff
@@ -74,6 +74,12 @@ envs:
     - name: "KIE_SERVER_CONTROLLER_TOKEN"
       example: "dasd373egds"
       description: "KIE server controller token for bearer authentication (Sets the org.kie.server.controller.token system property)"
+    - name: "LOG_LEVEL"
```

Comment:
also, please add some behave tests to test if the file is placed in the correct place.
if LOG_LEVEL is set to DEBUG, also set for debug messages in the smartrouter logs.

Reply:
ok

Reply:
still not seeing the behave tests on jboss-kie-modules.

Reply:
adding

Reply:
@spolti behave test added